tests/net/deckard/server.rs

```diff
@@ -75,8 +77,13 @@ fn do_adjust<Octs: Octets>(
         msg.push(rec).unwrap();
     }
     let mut msg = msg.additional();
-    for _a in &sections.additional {
-        todo!();
+    for a in &sections.additional {
+        let rec = if let ZonefileEntry::Record(record) = a {
```

Comment:
This can be simplified using a let-else pattern, e.g. `let ZonefileEntry::Record(rec) = a else { panic!() };`.

Reply:
if let ZonefileEntry::Record(rec) = a {} else {
            panic!("include not expected")
        };

gives:
error[E0425]: cannot find value `rec` in this scope
  --> tests/net/deckard/server.rs:84:18
   |
84 |         msg.push(rec).unwrap();
   |                  ^^^ not found in this scope

Reply:
It should be:
```rust
        let ZonefileEntry::Record(rec) = a else {
            panic!("include not expected")
        };
```